PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/WebAppConfiguration.java

```diff
@@ -341,4 +343,30 @@ public boolean isSlotPanelVisible() {
     public void setSlotPanelVisible(boolean slotPanelVisible) {
         webAppSettingModel.setSlotPanelVisible(slotPanelVisible);
     }
+
+    public void setModel(final WebAppComboBoxModel webAppComboBoxModel) {
+        setWebAppId(webAppComboBoxModel.getResourceId());
+        setWebAppName(webAppComboBoxModel.getAppName());
+        setResourceGroup(webAppComboBoxModel.getResourceGroup());
+        setSubscriptionId(webAppComboBoxModel.getSubscriptionId());
+        if (webAppComboBoxModel.isNewCreateResource()) {
+            setCreatingNew(true);
+            final WebAppSettingModel settingModel = webAppComboBoxModel.getWebAppSettingModel();
+            setCreatingResGrp(settingModel.isCreatingResGrp());
+            setCreatingAppServicePlan(settingModel.isCreatingAppServicePlan());
+            setAppServicePlanName(settingModel.getAppServicePlanName());
+            setRegion(settingModel.getRegion());
+            setPricing(settingModel.getPricing());
+            setAppServicePlanId(settingModel.getAppServicePlanId());
+            setOS(settingModel.getOS());
```

Comment:
make an util method  to copy all fields between WebAppComboBoxModel  and WebAppConfiguration